node/src/GlideClusterClient.ts

```diff
@@ -406,10 +408,9 @@ export class GlideClusterClient extends BaseClient {
      * ```
      */
     public ping(message?: string, route?: Routes): Promise<string> {
-        return this.createWritePromise(
-            createPing(message),
-            toProtobufRoute(route),
-        );
+        return this.createWritePromise(createPing(message), {
+            route: toProtobufRoute(route),
```

Comment:
decoder?

Reply:
it should be added to all commands which returns string. and it will be gradually.